packages/messaging/utils/types.ts

```diff
@@ -25,10 +25,12 @@ export type MessagePayload = UnknownObject &
 	IdentifiedPayload & {
 		from?: string;
 		type: RequestType;
-		messsage?: string;
+		requestId?: string;
+		message?: string;
 		transaction?: string;
 		options?: unknown;
 		passcode?: string;
+		isApproved?: boolean;
 	};
```

Comment:
should not include field like `isApprove`, `transactions` inside `@walless/messaging`.
MessagePayload should only cover enough data for generic "messaging", i.e `type`, `requestId` field.

Proposal for `MessagePayload` interface (pseudo example):
```typescript
type MessagePayload<T> = {
  from?: string;
  type: RequestType;
  data?: T;
}
```

While different modules communicate they need to extends `MessagePayload` into domain-specific type, e.g pseudo code:
```typescript
export type RequestConnectMessage = MessagePayload<{
  isApproved: boolean;
}>
```

Ultimate goal is, that inside Messaging module, it only care forwarding message with bare minimal interface from `MessagePayload`, and from messaging client it use Concrete data type, in which e.g they know for sure, `message.data.isApprove` always exist (static/accurate type).